addon/ng2/blueprints/component-test/index.js

```diff
@@ -1,12 +1,12 @@
 module.exports = {
   description: '',
-  
+
   // ******************************************************
   // ******************************************************
   // LEAVE THIS HERE
   // Must override install to prevent ember's component tests
   // ******************************************************
   // ******************************************************
-  
-  install: function(options){}
+
+  install: function(options) {}// eslint-disable-line no-unused-vars
```

Comment:
yes it still complains, I did not remove the argument because of the giant warning comment block. If you feel its safe to remove I will.